runtime-infra/pn-data-monitoring.yaml

```diff
@@ -173,6 +187,91 @@ Resources:
             Action:
               - "sts:AssumeRole" 
 
+  EC2DataMonitoringInstanceProfile:
+    Type: AWS::IAM::InstanceProfile
+    Condition: IsCoreAccount
+    Properties:
+      InstanceProfileName: pn-data-monitoring-ec2-instance-profile
+      Path: "/"
+      Roles:
+        - !Ref EC2DataMonitoringServiceRole
+  
+  EC2DataMonitoringServiceRole:
+    Type: AWS::IAM::Role
+    Condition: IsCoreAccount
+    Properties:
+      RoleName: pn-data-monitoring-ec2-service-role
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - "ec2.amazonaws.com"
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+
+  DataMonitoringEC2ManagedPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Condition: IsCoreAccount
+    Properties:
+      ManagedPolicyName: pn-data-monitoring-ec2-managed-policy
+      Roles:
+        - !Ref EC2DataMonitoringServiceRole
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          # lettura su tutto il bucket dei log e parquet
+          - Effect: Allow
+            Action:
+              - "s3:Get*"
+              - "s3:List*"
+            Resource: 
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ARN in Resource DataMonitoringEC2ManagedPolicy contains hardcoded Partition in ARN or incorrectly placed Pseudo Parameters